WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -820,6 +820,9 @@ public void onActionResult(CommentActionResult result) {
                         ToastUtils.showToast(getActivity(), getString(R.string.note_reply_successful));
                         mEditReply.setText(null);
                         mEditReply.getAutoSaveTextHelper().clearSavedText(mEditReply);
+
+                        // approve the comment
+                        moderateComment(CommentStatus.APPROVED);
```

Comment:
I think for this PR simply checking for the approved status is all that's needed. We may want a separate PR, though, to bring us in line with wp-admin.